modules/@apostrophecms/translation/index.js

```diff
@@ -54,7 +57,14 @@ module.exports = {
             );
           }
 
-          return module.translate(req, providerName, doc, source, target, options);
+          return manager.translate(req, {
```

Comment:
The signature should stay the same, additional data should be passed via the `options` object.